javaagent-tooling/src/main/java/io/opentelemetry/javaagent/tooling/muzzle/MuzzleVisitor.java

```diff
@@ -133,28 +133,18 @@ public MethodVisitor visitMethod(
     }
 
     public Reference[] generateReferences() {
-      // track sources we've generated references from to avoid recursion
```

Comment:
Why this is not required any more?

Reply:
Because it served no purpose from the start: because `adviceClassNames` always contained unique elements `referenceSources.contains(adviceClass)` always returned false.